debian/changelog

```diff
@@ -1,3 +1,11 @@
+cloud-init (23.4.1-0ubuntu1~23.10.1ubuntu1) UNRELEASED; urgency=medium
```

Comment:
This should be `23.10.2` rather than `1ubuntu1`. File a bug against new_upstream_snapshot.py if we need one.

Reply:
Also please bump `UNRELEASED` to `mantic` so we can release this